control-plane/connect-inject/peering_dialer_controller_test.go

```diff
@@ -251,7 +251,12 @@ func TestReconcile_CreateUpdatePeeringDialer(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 
 			// Create test consul server.
-			acceptorPeerServer, err := testutil.NewTestServerConfigT(t, nil)
```

Comment:
This change was due to a failure after updating the api module.

Reply:
I think there are a few more tests that need a similar update made to their fixtures.